### PR TITLE
Run3-gex145 Change getByLabel to use tokens in Alignment/OfflineValidation

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Change getByLabel to use tokens in Alignment/OfflineValidation

#### PR validation:

Use the runTheMatrix test workflows

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Review by ChrisMisan — COMMENTED** [review]
@bsunanda Could you add fillDescriptions as the part of the cleanup?

**Comment by ChrisMisan** [comment]
@bsunanda kind reminder

**Comment by bsunanda** [comment]
OK. I shall try to do this in coming days. Sorry I am out of station right now. I shall do that once I get back.
________________________________

**Comment by ChrisMisan** [comment]
@bsunanda just a reminder

**Comment by bsunanda** [comment]
Sorry - I shall take it up soon

File: Alignment/OfflineValidation/plugins/ValidationMisalignedTracker.cc [hunk 1]
```diff
@@ -220,15 +224,30 @@ ValidationMisalignedTracker::~ValidationMisalignedTracker() {
...
 // member functions
 //
+void ValidationMisalignedTracker::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  std::vector<edm::InputTag> labels;
+  std::vector<std::string> associators;
+  desc.add<std::vector<edm::InputTag> >("label", labels);
+  desc.add<edm::InputTag>("label_tp_effic", edm::InputTag("FinalTrackRefitter"));
+  desc.add<edm::InputTag>("label_tp_fake", edm::InputTag("TrackRefitter"));
+  desc.add<std::vector<std::string> >("associators", associators);
```
Comment (mmusich) [inline Alignment/OfflineValidation/plugins/ValidationMisalignedTracker.cc:234]:
```suggestion
  desc.add<std::vector<edm::InputTag> >("label", {});
  desc.add<edm::InputTag>("label_tp_effic", edm::InputTag("FinalTrackRefitter"));
  desc.add<edm::InputTag>("label_tp_fake", edm::InputTag("TrackRefitter"));
  desc.add<std::vector<std::string> >("associators", {});
```

declaring the `vector`s is unnecessary

**Comment by ChrisMisan** [comment]
please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ChrisMisan** [comment]
please test

- retest

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by mmusich** [comment]
@ChrisMisan 

> PR description doesn't explain the changes in comparison. Is the source of the difference understood?

I think, since the differences are very concentrated in [11634.7](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_13_0_X_2022-12-04-2300+d422f2/54515/dqm-histo-comparison-summary.html) that this is a case of https://github.com/cms-sw/cmssw/issues/39803.
This PR has very clearly nothing to do with it.

**Comment by ChrisMisan** [comment]
+alca

- differences related to the issue #39803

**Pull request merged**